portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,133 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <title>Samuel Sandoval</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="shortcut icon" href="images/SS_Initials.png" type="image/SS_initials"/>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
```

Comment:
I this needed, given these are the same as the css files you added above, and you include them at the end? Or did I miss something?

Reply:
Fixed that and just made that stylesheet get imported in the style.css